Storage/SQL/BrowserSchema.swift

```diff
@@ -232,155 +231,180 @@ open class BrowserSchema: Schema {
-            "cache_key LONGVARCHAR," +
-            "url TEXT," +
-            "title TEXT," +
-            "guid TEXT," +
-            "visitCount INTEGER," +
-            "visitDate DATETIME," +
-            "is_bookmarked INTEGER" +
-    ") "
+    let topSitesTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableCachedTopSites) (
+            historyID INTEGER,
+            url TEXT NOT NULL,
+            title TEXT NOT NULL,
+            guid TEXT NOT NULL UNIQUE,
+            domain_id INTEGER,
+            domain TEXT NO NULL,
+            localVisitDate REAL,
+            remoteVisitDate REAL,
+            localVisitCount INTEGER,
+            remoteVisitCount INTEGER,
+            iconID INTEGER,
+            iconURL TEXT,
+            iconDate REAL,
+            iconType INTEGER,
+            iconWidth INTEGER,
+            frecencies REAL
+        )
+        """
+
+    let pinnedTopSitesTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TablePinnedTopSites) (
+            historyID INTEGER,
+            url TEXT NOT NULL UNIQUE,
+            title TEXT,
+            guid TEXT,
+            pinDate REAL,
+            domain TEXT NOT NULL
+        )
+        """
+
+    let domainsTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableDomains) (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            domain TEXT NOT NULL UNIQUE,
+            showOnTopSites TINYINT NOT NULL DEFAULT 1
+        )
+        """
+
+    let queueTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableQueuedTabs) (
+            url TEXT NOT NULL UNIQUE,
+            title TEXT
+        )
+        """
+
+    let syncCommandsTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableSyncCommands) (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            client_guid TEXT NOT NULL,
+            value TEXT NOT NULL
+        )
+        """
+
+    let clientsTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableClients) (
+            guid TEXT PRIMARY KEY,
+            name TEXT NOT NULL,
+            modified INTEGER NOT NULL,
+            type TEXT,
+            formfactor TEXT,
+            os TEXT,
+            version TEXT,
+            fxaDeviceId TEXT
+        )
+        """
+
+    let tabsTableCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableTabs) (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            client_guid TEXT REFERENCES clients(guid) ON DELETE CASCADE,
+            url TEXT NOT NULL,
+            title TEXT,
+            history TEXT,
+            last_used INTEGER
+        )
+        """
+
+    let activityStreamBlocklistCreate = """
+        CREATE TABLE IF NOT EXISTS \(TableActivityStreamBlocklist) (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            url TEXT NOT NULL UNIQUE,
+            created_at DATETIME DEFAULT CURRENT_TIMESTAMP
+        )
+        """
+
+    let pageMetadataCreate = """
+        CREATE TABLE IF NOT EXISTS \(TablePageMetadata) (
```

Comment:
I am slightly tempted to suggest dropping this idiom for table names inside queries (keeping them for parameters, until switching to enums!). We should never rename them, and if we do we will need to change queries, too. Thoughts?

Reply:
I'm in complete agreement. Should I do it in this PR?

Reply:
Thumbs up from me.